Acrolinx.Sidebar/ISidebar.cs

```diff
@@ -187,12 +189,67 @@ public bool SupportCheckSelection
 
     public class CheckedEventArgs : CheckEventArgs
     {
-        internal CheckedEventArgs(string checkId, Range range) : base(checkId)
+        internal CheckedEventArgs(string checkId, Range range, JArray embedCheckInformation, string inputFormat) : base(checkId)
         {
             Range = range;
+            EmbedCheckInformation = new JObject();
+            EmbedCheckInformation["embedCheckInformation"] = embedCheckInformation;
+            if (inputFormat.ToUpper().Equals("XML"))
+                InputFormat = Format.XML;
+            else if (inputFormat.ToUpper().Equals("HTML"))
+                InputFormat = Format.HTML;
+            else if (inputFormat.ToUpper().Equals("MARKDOWN"))
+                InputFormat = Format.Markdown;
+            else if (inputFormat.ToUpper().Equals("TEXT"))
+                InputFormat = Format.Text;
+            else if (inputFormat.ToUpper().Equals("WORD_XML"))
+                InputFormat = Format.Word_XML;
+            else InputFormat = Format.Auto;
```

Comment:
Is this true? I assume that if it is not returned from the server we might talk to an older API. In that case it might be the format we've set while submitting the check.